src/main/scala/cognite/spark/v1/DefaultSource.scala

```diff
@@ -390,6 +410,11 @@ object DefaultSource {
   def saveTracingHeaders[F[_]: Functor: Trace](): F[Seq[(String, String)]] =
     Trace[F].kernel.map(saveTracingHeaders(_))
 
+  def logParamterOrDefault[A](name: String, value: Option[A], default: => A): A = value match {
```

Comment:
Typo in the name
Also, this function also returns the value.. Below, when calling it, it looks weird that we are assigning a logging function 🤔